### PR TITLE
Fix long blockquotes wrapping

### PR DESCRIPTION
**Pull request by hsmett**
Hello !
Here is an attempt to fix wrapping issues within blockquote.
I don't know if this is the good way to go, I would love to get some feedback @Alir3z4 :)
Thanks

### CODE REVIEW

File: html2text/__init__.py [hunk 1]
```diff
@@ -905,6 +905,8 @@ def optwrap(self, text):
                     indent = ""
                     if para.startswith("  " + self.ul_item_mark):
                         indent = "    "  # For list items.
+                    elif para.startswith("> "):
```
Comment (Alir3z4) [inline html2text/__init__.py]: If you could put some comments here and describe the reasoning as comment, it would be really really good.

Comment (hsmett) [reply html2text/__init__.py]: Is that ok ?

**Comment by hsmett** [comment]
Does this PR meet your expectations? :)

File: test/break_preserved_in_blockquote.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 a
 
 > b  
-> c
+> c
```
Comment (jdufresne) [inline test/break_preserved_in_blockquote.md]: Why does this result now have trailing whitespace? I wouldn't expect this test to change.

Comment (hsmett) [reply test/break_preserved_in_blockquote.md]: Indeed, the current code makes all blockquotes to end with a linebreak.

This behavior comes from the handling of list-items wrapping:
https://github.com/Alir3z4/html2text/blob/master/test/wrap_list_items_example.md

Those trailing whitespaces don't seem very logical to me neither, or I am missing something?

**Comment by jdufresne** [comment]
> Those trailing whitespaces don't seem very logical to me neither, or I am missing something?

I agree. If removing them does not cause new unexpected test failures, I suggest we remove the trailing space..

**Comment by hsmett** [comment]
This seems to do the job.

File: html2text/__init__.py (hunk 1 above)
Comment (Alir3z4) [reply html2text/__init__.py]: Yes, it does!

**Comment by jdufresne** [comment]
This looks good to me. @Alir3z4 thoughts on merging this and cutting a release?

**Comment by Alir3z4** [comment]
Oh, I missed this for the release.
No worries,

We can merge this and do another release any other day.

**Comment by hsmett** [comment]
Thank you!

**Pull request merged**